app/mci/cmd/create.go

```diff
@@ -106,8 +108,10 @@ func AddFlags(cmd *cobra.Command, options *CreateOptions) error {
 	cmd.Flags().StringVarP(&options.IngressFilename, "ingress", "i", options.IngressFilename, "filename containing ingress spec")
 	cmd.Flags().StringVarP(&options.Kubeconfig, "kubeconfig", "k", options.Kubeconfig, "path to kubeconfig")
 	cmd.Flags().StringVarP(&options.GCPProject, "gcp-project", "", options.GCPProject, "name of the gcp project")
-	// TODO Add a verbose flag that turns on glog logging.
 	cmd.Flags().BoolVarP(&options.ForceUpdate, "force", "f", options.ForceUpdate, "overwrite existing settings if they are different.")
+	// TODO Add a verbose flag that turns on glog logging, or figure out how
+	// to accept glog flags in addition to the cobra flags.
+	//flag.Set("logtostderr", "true")
```

Comment:
remove. Does uncommenting this make a difference? (Asking since this is already set in cmd.go)